Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM ubuntu:16.04
+FROM ubuntu:latest
```

Comment:
should this one be locked to 20.04 given it's using the system python? Very possible that the locked python library versions won't be compatible with newer version of python

Reply:
I'll defer to your better judgment!